dev-tools/packer/docker/xgo-image-deb6/go-1.8.3/Dockerfile

```diff
@@ -9,7 +9,7 @@ MAINTAINER Tudor Golubenco <tudor@elastic.co>
 
 # Configure the root Go distribution and bootstrap based on it
 RUN \
-  export ROOT_DIST="https://storage.googleapis.com/golang/go1.8.3.linux-amd64.tar.gz" && \
+  export ROOT_DIST="https://storage.googleapis.com/golang/go1.9.2.linux-amd64.tar.gz" && \
   export ROOT_DIST_SHA1="838c415896ef5ecd395dfabde5e7e6f8ac943c8e" && \
```

Comment:
Pretty sure the SHA1 sums need updated too.

Reply:
good point. updated both. Also renamed the directory which is called `1.8.3`.